arkane/gaussian.py

```diff
@@ -468,3 +468,30 @@ def loadNegativeFrequency(self):
             raise Exception('Unable to find imaginary frequency of {1} '
                             'in Gaussian output file {0}'.format(self.path, self.species.label))
         return frequency
+
```

Comment:
We are not using this because, as you mentioned, we removed the commit to print out level of theory. I will remove this commit too. I initially left it because I am thinking of fixing it, adding a method to find basis set, and do the same for energy calculations, if/when I have time.